src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  margin: auto;
+  padding: 50px $hor-padding $hor-padding;
+  max-width: ($cell-size*$cell-per-row)+
+    ($gap*$cell-per-row - $gap)+
+    ($hor-padding*2);
```

Comment:
it's better to use function here instead of such calculation here
If you haven't worked with sass functions - feel free to ask at fe_chat